docs/debt/debt-436-stale-spec-docs-claim-user-created-webhook-handling.md

```
@@ -0,0 +1,54 @@
+
+**Status:** Open
+**Priority:** P3
+**Date:** 2026-06-30
+
+---
+
+## Description
+
+`docs/specs/master_spec.md` and `docs/specs/master_spec_part2.md` both still list, under "Events handled" for the Clerk webhook:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)